Drop C(XX)_STANDARD and use newer Cmake version

Åke Sandgren, 2026-10-29
diff -ru gcta-1.94.1.orig/CMakeLists.txt gcta-1.94.1/CMakeLists.txt
--- gcta-1.94.1.orig/CMakeLists.txt	2026-10-29 08:33:38.573608155 +0100
+++ gcta-1.94.1/CMakeLists.txt	2026-10-29 09:22:36.126947502 +0100
@@ -1,10 +1,8 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+CMAKE_MINIMUM_REQUIRED(VERSION 4.0)
 SET(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 SET(PROJECT_NAME_STR GCTA2)
 PROJECT(${PROJECT_NAME_STR} C CXX)
 SET(CMAKE_INCLUDE_CURRENT_DIR ON)
-SET(CMAKE_CXX_STANDARD 11)
-SET(CMAKE_C_STANDARD 11)
 
 # get system architecture
 EXECUTE_PROCESS(COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCHITECTURE)
